app/models/moment_template.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: moment_templates
+#
+#  id          :bigint           not null, primary key
+#  name        :string
+#  description :text
+#  slug        :string
+#  user_id     :integer
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+class MomentTemplate < ApplicationRecord
+  extend FriendlyId
+
+  friendly_id :name
+  validates :user_id, :name, :description, presence: true
+  belongs_to :user
```

Comment:
I'm curious, base on the MomentTemplate factory, we could have a MomentTemplate without a user, right? If that's the case, we'd probably need to set the following:

```suggestion
  belongs_to :user, optional: true
```

`belongs_to` since rails 5 requires an ID by default, but if we want this to not be `nil`, disregard this message.